core-rules/src/main/java/org/hisp/dhis/rules/RuleVariableValueMapBuilder.java

```diff
@@ -96,12 +96,12 @@ static RuleVariableValueMapBuilder target(@Nonnull RuleEnrollment ruleEnrollment
     }
 
     @Nonnull
-    static RuleVariableValueMapBuilder target(@Nonnull RuleEvent ruleEvent) {
+    public static RuleVariableValueMapBuilder target(@Nonnull RuleEvent ruleEvent) {
```

Comment:
We should not change this method to public because It's [private to package](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html). The goal is not to access from outside from package to this class,  only RuleEngine, RuleEngineContext ... can access it. If we change it to public we are breaking encapsulation.